### PR TITLE
feat: tekton builds for azure

### PR DESCRIPTION
**Pull request by kseniyashaydurova**
I created a new branch with tekton builds for `azure`, it was created from `main` branch with some changes from previous `azure` branch and some fixes

### CODE REVIEW

**Review by populov — APPROVED** [review]

File: tekton-pipelines/buildpacks-frontend/triggers/build-pipeline-trigger.yaml [hunk 1]
```diff
@@ -41,7 +41,7 @@ spec:
         - taskName: kustomize
           serviceAccountName: $(tt.params.application)-build-pipeline-kustomize-sa
       pipelineRef:
-        name: frontend-buildpack-build-pipeline
+        name: buildpack-frontend-build-pipeline
```
Comment (dmitry-mightydevops) [inline tekton-pipelines/buildpacks-frontend/triggers/build-pipeline-trigger.yaml:44]: Let's use same names we have in our infra

```
➜ k get pipeline -n ci
NAME                                AGE
buildpack-build-pipeline            46d
dotnet-buildpack-build-pipeline     13d
frontend-buildpack-build-pipeline   30d
kaniko-build-pipeline               66d
wordpress-build-pipeline            41d
```

So we need `frontend-buildpack-build-pipeline `

**Review by dmitry-mightydevops — CHANGES_REQUESTED** [review]

**Review by dmitry-mightydevops — CHANGES_REQUESTED** [review]

File: tekton-pipelines/buildpacks-frontend/triggers/build-pipeline-trigger.yaml (hunk 1 above)
Comment (kseniyashaydurova) [reply tekton-pipelines/buildpacks-frontend/triggers/build-pipeline-trigger.yaml:44]: @dmitry-mightydevops that is the name from current `main` branch with your last changes in `main` for frontend buildpack, We need to actualize pipelines in `saritasa-devops-tekton` repo and `saritasa-rocks-cluster` and release a new version. By the way, do we have more projects in client infra, which uses `buildpack-frontend-build-pipeline` except mine?

Comment (dmitry-mightydevops) [reply tekton-pipelines/buildpacks-frontend/triggers/build-pipeline-trigger.yaml:44]: actually let's keep then as it was i.e. `buildpack-TYPE`, so the prefix is the `buildpack`, I will adjust then saritasa-rocks & cloud to use that naming so we're in sync.

**Review by dmitry-mightydevops — APPROVED** [review]

**Comment by kseniyashaydurova** [comment]
Closed PR, it is not merged, cause there are overridden buildpack tasks for Azure and it shouldn't go into `main` branch.

Currently it is supposed that tekton pipelines for Azure can be used by `0.24.2-azure` tag, which leads to `feature/azure-buildpack` branch.

**Pull request closed without merging**